### ISSUE TITLE
utime doesn't exist for Snek, but time does

### ISSUE THREAD

**Issue by mobluse** [BIPES/BIPES#70]
The module `utime` doesn't exist for Snek, but `time` does. That means that `delay` in BIBES doesn't work with Snek. For Microbit `time` is an alias for `utime`, so it might be possible to change to `time` for all boards.

**Comment by rafaelaroca** [comment]
Hello @mobluse!

I agree with you: utime does not exit for Snek. But please note that when you select the Arduino/Snek, the delay block is modified and will generate " time.sleep(1) " instead of utime.sleep(). Your  point is to change all calls to utime to time? 

Here is an example that will generate the correct code:

http://bipes.net.br/beta2/ui/#a6kegm

regards
Rafael

**Comment by mobluse** [comment]
Yes, I see it's right in your example, but if I change device from Arduino to BBC Microbit time doesn't change to utime. It might have been the same situation in my project where I discovered it: that I developed it initially for Microbit and when I changed to Arduino, utime didn't change to time. I think the generated code should change so that it works on the selected device. It could be easy to accomplish this if one use the general module names like time instead of utime.
